portfolio/pom.xml

```diff
@@ -35,7 +42,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```

Comment:
Can this TODO be deleted now?

Reply:
I hadn't noticed it, so thank you for letting me know!